test/HydraDelegation/VestedDelegation.test.ts

```diff
@@ -651,9 +651,10 @@ export function RunVestedDelegationTests(): void {
           vestManagerOwner,
           liquidToken,
           vestManager.address,
-          delegatedBalance,
+          await hydraDelegation.calculateOwedLiquidTokens(vestManager.address, delegatedBalance),
           DEADLINE
         );
+
         await vestManager.cutVestedDelegatePositionWithPermit(
```

Comment:
it would be nice to add a test for calculating that the right % is cut for a week